Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -85,6 +85,10 @@ private class TabCell: UITableViewCell {
         setupFrames()
 
         self.animator.originalCenter = CGPoint(x: UIScreen.mainScreen().bounds.width / 2, y: TabTrayControllerUX.CellHeight / 2)
+
+        self.accessibilityCustomActions = [
+            UIAccessibilityCustomAction(name: NSLocalizedString("Close", comment: ""), target: self.animator, selector: "closeWithoutGesture")
```

Comment:
Add a comment. Also, we always name selectors starting with "SEL" so they're easy to identify.